sdks/python/apache_beam/io/gcp/gcsio.py

```diff
@@ -155,7 +156,10 @@ def __init__(self, storage_client=None):
           credentials=auth.get_service_credentials(),
           get_credentials=False,
           http=get_new_http(),
-          response_encoding='utf8')
+          response_encoding='utf8',
+          additional_http_headers={
+              "User-Agent": "apache-beam-%s" % apache_beam.__version__
+          })
```

Comment:
Will it be possible to add a unit test to make sure this stays consistent and does not get broken ?

Reply:
added tests for BQ and GCS